parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs

```diff
@@ -165,10 +164,10 @@ pub type Barrier = DenyThenTry<
 		AllowKnownQueryResponses<PolkadotXcm>,
 		WithComputedOrigin<
 			(
-				// Allow anything to pay for execution.
+				// If the message is one that immediately attemps to pay for execution, then allow it.
 				AllowTopLevelPaidExecutionFrom<Everything>,
-				// Parent and its exec plurality get free execution.
-				AllowExplicitUnpaidExecutionFrom<ParentOrParentsExecutivePlurality>,
+				// Parent and its pluralities (i.e. governance bodies) get free execution.
+				AllowExplicitUnpaidExecutionFrom<ParentOrParentsPlurality>,
```

Comment:
Why don't we have `Fellowship` here as well as on the Polkadot hub?

Reply:
I did it according to statemine, but I think it is related to the fact that Kusama does not have Collectives parachain, polkadot's `FellowsPlurality` points to the Collectives parachain [here](https://github.com/paritytech/cumulus/pull/2449/files#diff-167b6926f30b1edd29856107ea200c33c40ddea60c85128cfe175e6c3749860bR114-R115)